EmployeeDirectory/Views/Role.cs

```diff
@@ -98,7 +107,7 @@ public void DisplayRoleList()
                     }
                 }
             }
-            catch (JsonException ex)
+            catch (Exception ex)
```

Comment:
If you have "Try-Catch" method in outer scope of a same method. Try to utilize it.